test/Sema/moveonly_resilient_type.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-swift-frontend -typecheck %s -enable-library-evolution -verify
+
```

Comment:
It'd be good to expect these same errors in thie file even when not passing `-enable-library-evolution` or not, since `isResilient` isn't suppose to give a different answer for public types whether the flag is there or not.